### DISCUSSION TITLE
Parallel fetching get Internal Server Error

### DISCUSSION THREAD

**Discussion by indrazm** [pocketbase/pocketbase#771] [Q&A] [answered]
Hey there.
So currently i am building https://indiemakers.id and i love pocketbase!

Here is the issue scenario.
I have a lounge page, which render component based on threads array data.
And in each thread component i fetch viewsData and repliesData based on threadId. 

I think it runs simoustanly, and the onlyone that works is the component that rendered in the last one.
And other than that will give me an error Internal server 500.

![image](https://user-images.githubusercontent.com/114550938/195869903-252e1934-3fc1-4722-b778-14a4e916a422.png)

Would you guys help me to fix this ?
Many thanks! :)

**Comment by ganigeorgiev** [comment, marked as answer]
Are you sure that it is throwing 500 error?

This seems like a result of the auto cancellation but the response certainly shouldn't be 500.
For more information on the auto cancellation and how to disable/tweak it - https://github.com/pocketbase/js-sdk#auto-cancellation.

If the above doesn't solve your issue, please provide a code sample how you are sending the requests or at least screenshot of the response from the browser "Inspect tool > Network" tab.

**Comment by indrazm** [comment]
Gosh, a huge thanks for quick answer!

It works! 
![image](https://user-images.githubusercontent.com/114550938/195872319-fd0caf32-bef1-470f-a28d-6fbe208c3398.png)
